design/KruizeConfiguration.md

```diff
@@ -139,3 +139,7 @@ The following environment variables are set using the `kubectl apply` command wi
     - Value: "true"
     - Details: This flag is added for getting the details of the inputs passed to the APIs and the corresponding response 
       generated by it. This helps us in debugging the API easily in case of failures.
+- **isROSEnabled**
+  - Description:  if set to True, the ROS application will use the old architecture, where ROS handles experiment creation, result updates, and recommendation updates. If set to False, ROS will utilize the Bulk API, allowing Kruize to manage experiment creation and generate recommendations on its behalf.
```

Comment:
@msvinaykumar - Please update all the remote monitoring tests to enable this. Also add new tests to test RM & LM.

Reply:
done